deployment/ansible/roles/raster-foundry.jenkins/tests/test_jenkins.py

```diff
@@ -12,6 +14,32 @@ def AnsibleDefaults(Ansible):
 
 
 @pytest.fixture()
+def JenkinsRoleDefaults(Ansible):
+    """ Load default variables into dictionary.
+
+    Args:
+        Ansible - Requires the ansible connection backend.
+    """
+    def GetJenkinsDefaultsFile():
+        """ Build azavea.jenkins filepath based on molecule's config """
+        molecule_cfg = ConfigParser()
+        molecule_cfg.optionxform = str
+        molecule_cfg.read("./molecule.cfg")
+        section = molecule_cfg.items("defaults")
+        defaults = {k: v for k, v in section}
+        jenkins_role_path = filter(lambda x:
+                                   os.path.exists(
+                                       os.path.join(x, "azavea.jenkins")),
+                                   defaults['roles_path'].split(":"))[0]
+        jenkins_defaults_file = os.path.join(jenkins_role_path,
```

Comment:
Since we know were things are relative to where this file lives, I think we can determine the path using `os.path.realpath(__file__)` and `os.path.dirname`.

```python
import os

print(os.path.realpath(__file__))
print(os.path.dirname(os.path.realpath(__file__)))
```

```bash
❯ python joker.py
/Users/hector/Desktop/joker.py
/Users/hector/Desktop
```